### PR TITLE
Removed the logic to trigger validation only after the user submits from the Form component

### LINKED ISSUE #1710: Remove the logic to trigger validation only after the user submits from the `Form` component
Ref: https://github.com/bigbinary/neeto-ui/pull/1707#issuecomment-1559661408

### PR DESCRIPTION
**Pull request by josephmathew900**
- Fixes #1710 

**Description**

**Checklist**

- ~[ ] I have made corresponding changes to the documentation.~
- ~[ ] I have updated the types definition of modified exports.~
- [x] I have verified the functionality in some of the neeto web-apps.
- ~[ ] I have added tests that prove my fix is effective or that my feature works~
- [ ] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by josephmathew900** [comment]
@Amaljith-K _a please review.

File: src/components/formik/Form/FormWrapper.jsx [hunk 1]
```diff
@@ -36,23 +21,20 @@ const FormWrapper = forwardRef(
...
+        try {
+          const errors = await validateForm();
+
+          if (Object.keys(errors).length > 0) {
+            setErrors(errors);
+            setTouched(errors);
+          } else {
+            onSubmit(values, formikBag);
+          }
+        } catch {}
```
Comment (Amaljith-K) [inline src/components/formik/Form/FormWrapper.jsx:37]: Is it ideal to keep this catch block empty?

**Review by Amaljith-K — APPROVED** [review]
@josephmathew900 _a rest of all LGTM

**Comment by josephmathew900** [comment]
@amaldinesh7 _a please review.

**Pull request merged**